internal/client/blueprints_test.go

```diff
@@ -63,6 +63,42 @@ func TestPostEmptyTOMLBlueprintV0(t *testing.T) {
 	require.False(t, resp.Status, "did not return an error")
 }
 
+// POST a TOML blueprint with an empty name
+func TestPostEmptyNameTOMLBlueprintV0(t *testing.T) {
+	// Use a blueprint with an empty Name
+	bp := `
+		name=""
+		version="0.0.1"
+		[package]
+		name="bash"
+		version="*"
+		`
+	resp, err := PostTOMLBlueprintV0(testState.socket, bp)
+	require.NoError(t, err, "failed with a client error")
+	require.False(t, resp.Status, "did not return an error")
+	require.False(t, resp.Status, "wrong Status (true)")
```

Comment:
What is the point of 2 sequential `require.False` both checking `resp.Status` ? In case `resp.Status` isn't False the test will abort on line 78 and line 79 will never be executed.

Reply:
In some of the other tests below you have only the 2nd require, not the 1st one.

Reply:
It makes sure there isn't a race condition with the universe failing inbetween the tests ;)

Or more likely it's a copy-paste error, good catch!